.pre-commit-config.yaml

```diff
@@ -5,17 +5,23 @@ repos:
   - id: check-yaml
   - id: end-of-file-fixer
   - id: trailing-whitespace
+  - id: check-case-conflict
+  - id: check-merge-conflict
+  - id: detect-private-key
+  - id: debug-statements
+  - id: check-added-large-files
 
 - repo: https://github.com/ambv/black
   rev: 20.8b1
   hooks:
   - id: black
 
-- repo: https://github.com/pycqa/pylint
-  rev: '2.6'
-  hooks:
-  - id: pylint
-    additional_dependencies: [pylint-quotes]
+# Currently disabled, as dependencies are managed through conda.
```

Comment:
But conda only takes care of installing pylint, but not of running as a pre-commit hook. Why removing it here?

Reply:
Sorry, that comment was a bit obscure :). pylint also checks the Python imports. In this case it complains e.g. about `kubernetes` not being known, because that's not listed as one of the pylint dependencies. It would be awesome if we could use the same conda environment file that's already used for the GitHub workflow to also perform the pre-commit checks.

Reply:
Updated all of this with your new configs now :).